test/debugInfo.fromasm.clamp.no-opts.map

```diff
@@ -1 +1 @@
-{"version":3,"sources":["tests/hello_world.c","tests/other_file.cpp","return.cpp","even-opted.cpp","fib.c","/tmp/emscripten_test_binaryen2_28hnAe/src.c","(unknown)"],"names":[],"mappings":"yLAIA,IACA,ICyylTA,aC7vlTA,OAkDA,0BCnGA,OACA,OACA,uBCAA,4BAKA,QAJA,OADA,8CAKA,0ICsi1DA,MCrvyDA"}
```

Comment:
TBH I have no idea what this change means.

Reply:
I think it means that things moved around - the location of code is not the same as before. That's surprising, I think?

Reply:
Aha, this is because I changed the element section to only be emitted if it has any elements. That moved the code section up in the file.